cs-excel/src/main/java/io/cloudslang/content/excel/services/ExcelServiceImpl.java

```diff
@@ -0,0 +1,356 @@
+        if (StringUtils.isBlank(format)) {
+            throw new ExcelOperationException(BAD_EXCEL_FILE_MSG);
+        } else
+            return (format.equalsIgnoreCase(FORMAT_XLSX)
+                    || format.equalsIgnoreCase(FORMAT_XLS)
+                    || format.equalsIgnoreCase(FORMAT_XLSM));
+
+    }
+
+    public static void updateWorkbook(final Workbook workbook, final String fileName) throws IOException {
+        try (FileOutputStream output = new FileOutputStream(fileName)) {
+            workbook.write(output);
+        } catch (IOException e) {
+            throw e;
+        }
+    }
+
+    public static String getFileFormat(final String excelFileName) {
+        final int index = excelFileName.lastIndexOf(".");
+        if (index > -1 && index < excelFileName.length()) {
+            return excelFileName.substring(index + 1);
+        }
+
+        return null;
+    }
+
+    public static Workbook getExcelDoc(final String excelFileName) throws ExcelOperationException,InvalidFormatException,IOException{
+        Workbook excelDoc = null;
+            excelDoc = getWorkbook(excelFileName);
+            if (excelDoc == null) {
+                throw new ExcelOperationException("Could not open " + excelFileName);
+            }
+        return excelDoc;
+    }
+
+    public static Sheet getWorksheet(final Workbook excelDoc, final String sheetName) throws ExcelOperationException {
+        final Sheet worksheet = excelDoc.getSheet(sheetName);
+        if (worksheet == null) {
+            throw new ExcelOperationException("Worksheet " + sheetName + " does not exist.");
+        }
+        return worksheet;
+    }
+
+    /**
+     * Get the type of a cell, it can be num, date, time or string
+     *
+     * @param cell
+     * @return
+     */
+    public static String getCellType(final Cell cell) {
+        final String result;
+        final double cellValueNumeric;
+        final CellType cellType = cell.getCellType();
+
+        if (cellType == CellType.NUMERIC) {
+            cellValueNumeric = cell.getNumericCellValue();
+
+            //date cell, it can be date, time or datetime
+            if (DateUtil.isCellDateFormatted(cell)) {
+                //time cell
+                if (cellValueNumeric < 1) {
+                    result = "time";
+                }
+                //date cell
+                else {
+                    result = "date";
+                }
+            }
+            //numeric cell
+            else {
+                result = "num";
+            }
+        }
+        //String cell
+        else {
+            result = "string";
+        }
+        return result;
+    }
+
+    /**
+     * compare the numeric values
+     *
+     * @param value1
+     * @param value2
+     * @param operator
+     * @return true or false
+     * @throws Exception
+     */
+    public static boolean compareNumericValue(double value1, double value2, final String operator) {
```

Comment:
I suggest to use Java out of the box compare options rather than implementing compare... methods